include/fmt/chrono.h

```diff
@@ -1081,6 +1119,26 @@ void write_floating_seconds(memory_buffer& buf, Duration duration) {
       num_fractional_digits);
 }
 
+template <typename Duration>
+void write_floating_seconds(memory_buffer& buf, Duration duration,
+                            int precision) {
+  if (precision < 0) {
+    write_floating_seconds(buf, duration);
+    return;
+  }
+
+  FMT_ASSERT(std::is_floating_point<typename Duration::rep>::value, "");
```

Comment:
Let's add an alias for `typename Duration::rep` and use it here and in several places below.

Reply:
Done